.travis.yml

```diff
@@ -1,3 +1,5 @@
+dist: trusty
```

Comment:
Travis CI was failing, after researching a bit I found that `dist: trusty` is needed if `xvfb` is used directly.

Doc: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-directly